lxd/db/cluster/devices.mapper.go

```diff
@@ -118,7 +119,7 @@ func GetDevices(ctx context.Context, tx *sql.Tx, parent string, filters ...Devic
 		}
 
 		if len(entries) == 0 {
-			return nil, fmt.Errorf("Cannot filter on empty DeviceFilter")
+			return nil, errors.New("Cannot filter on empty DeviceFilter")
```

Comment:
These files are generated by `make update-schema` - It might be worth updating https://github.com/canonical/lxd/blob/300a2f7af72ff99d0735fa1417dccf7317162d4c/lxd/db/generate/db/method.go#L317 to use `errors.New` and adding `errors` to the list of imports here https://github.com/canonical/lxd/blob/300a2f7af72ff99d0735fa1417dccf7317162d4c/lxd/db/generate/db/constants.go#L6

Then running `make update-schema`, and then running the `perfsprint` tool. Otherwise these will revert when schema generation is run again.

Reply:
good spot!

Reply:
I've just realised that the update I've commented on is a different error to the one I found in `lxd-generate` so presumably there are a few of these to fix first.

Reply:
I only found a few occurrences so just fixed it as I went. Here's the PR https://github.com/canonical/lxd/pull/15507

Reply:
Ah thanks for catching and fixing it properly! I'll try and be more careful with future mechanical changes :)